code/SecurityAdmin.php

```diff
@@ -276,9 +314,13 @@ public function GroupImportForm()
 
     /**
      * Disable GridFieldDetailForm backlinks for this view, as its
+     * @deprecated 1.13.0 Will be removed without equivalent functionality to replace it
      */
     public function Backlink()
```

Comment:
I think we made a mistake by removing this - see `GridFieldDetailForm_ItemRequest::getBackLink()` - we should probably still explicitly exclude security admin from the back link functionality (I'm presuming the reason it was excluded originally is still valid)

Reply:
I don't understand what you're suggesting we do?

It's removed in CMS5 at this point? So you're saying we should add it back it?  Do we know if it's broken? Is there a card to add BackLink back in?

Reply:
Nevermind easier to just keep it gone and deprecate.